packages/ui/components/LogoType.tsx

```
@@ -0,0 +1,47 @@
+export default function LogoType({ className }: { className?: string }) {
+  return (
+    <svg
+      className={className}
+      viewBox="0 0 317 52"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)